__tests__/app.test.js

```diff
@@ -167,4 +202,50 @@ describe("/api/review/review_id/comments", () => {
         expect(res.body.msg).toBe("bad request");
       });
   });
-});
\ No newline at end of file
+});
+
+describe("POST /api/review/review_id/comments", () => {
+  test("POST - 201: add a comment to the database and respond with newly created comment", () => {
+    const newComment = {
+      body: "WOW best game this year!",
+      username: "mallionaire",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toMatchObject({
+          comment_id: 7,
+          body: "WOW best game this year!",
+          votes: 0,
+          author: "mallionaire",
+          review_id: 1,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("ERR - 400: when no info passed, respond with 400", () => {
+    const newComment = {};
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("bad request");
+      });
+  });
+  test("ERR - 404 no review available for comment post", () => {
+    const newComment = {
+      body: "WOW best game this year!",
+      username: "mallionaire",
+    };
+    return request(app)
+      .post("/api/reviews/99/comments")
+      .send(newComment)
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("invalid review id");
+      });
+  });
```

Comment:
Great testing so far but needs a couple more:
- a reviewID with an invalid id (e.g id banana)
- A username that doesnt exist on the POST request
- Can it ignore unnecessary properties on the newComment object?